### PR TITLE
Addressed the rules for config.allowUnsafeUpsert

### PR DESCRIPTION
**Pull request by jmaicaaan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -332,7 +332,7 @@ function cms(loopbackApplication, options) {
 
   if (config.allowUnsafeUpsert) {
```
Comment (carlitolee) [inline index.js]: I think the property allowUnsafeUpsert should be part of the public object.

Comment (tma-isbx) [inline index.js]: Other projects have been using it in the current location. Also, we probably don't want to expose that to the client side so explicitly.

**Comment by tma-isbx** [comment]
I think Carl and I talked, and I think he is agreeable that it should not be under config.public.

File: index.js [hunk 2]
```diff
@@ -332,7 +332,7 @@ function cms(loopbackApplication, options) {
...
     console.warn('Warning (isbx-loopback-cms): /model/save end point is running in compatibility mode, please update your ACL rules asap then remove config.allowUnsafeUpsert or set it to false');
```
Comment (csapinoso) [inline index.js]: @jmaicaaan please update the warning to say:

> 'Warning (isbx-loopback-cms): /model/save end point is running in compatibility mode, please update your ACL rules asap then set config.allowUnsafeUpsert to false'

**Pull request merged**